File: Linguagem C Exercicios/C-Condicional/NOTAS.c
#include <stdio.h>
#include <stdlib.h>

int main()
{

    double n1,n2,soma = 0;

    printf("Digite a primeira nota: ");
    scanf("%lf",&n1);
    printf("Digite a segunda nota: ");
    scanf("%lf",&n2);

    soma = n1 + n2 ;
    printf("NOTA FINAL = %.1lf\n",soma);

    if (soma >= 60)
    {

        printf("APROVADO !!");
    }
    else
    {
        printf("REPROVADO");
    };


    return 0;
}

File: Linguagem Java/Exercicios/Notas.java
import java.util.Locale;
import java.util.Scanner;

public class Notas {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		double n1,n2,notaFinal;
		
		System.out.print("Digite a primeira nota: ");
		n1 = sc.nextDouble();
		System.out.print("Digite a segunda nota: ");
		n2 = sc.nextDouble();
		
		notaFinal = n1+n2;
		
		if (notaFinal >= 60.00) {
			System.out.println("APROVADO");
			
		}else {
			System.out.println("REPROVADO");			
		};
		
		sc.close();

	}

}
